src/images/kafka.rs

```diff
@@ -0,0 +1,124 @@
+use crate::{core::WaitFor, Image};
+use std::{collections::HashMap, path::PathBuf};
+
+const CONTAINER_IDENTIFIER: &str = "confluentinc/cp-kafka";
+const DEFAULT_TAG: &str = "6.1.1";
+
+pub const KAFKA_PORT: u16 = 9093;
+const ZOOKEEPER_PORT: u16 = 2181;
+
+#[derive(Clone, Debug)]
+pub struct KafkaArgs {
+    pub bash_pathname: PathBuf,
+    pub options: Vec<String>,
+    pub commands: String,
+}
+
+impl Default for KafkaArgs {
+    fn default() -> Self {
+        let starter_commands = format!(
+            "echo 'clientPort={}' > zookeeper.properties; \
+echo 'dataDir=/var/lib/zookeeper/data' >> zookeeper.properties; \
+echo 'dataLogDir=/var/lib/zookeeper/log' >> zookeeper.properties; \
+zookeeper-server-start zookeeper.properties & \
+export KAFKA_ZOOKEEPER_CONNECT='localhost:{}'; \
+export KAFKA_ADVERTISED_LISTENERS='PLAINTEXT://localhost:{},BROKER://localhost:9092'; \
+. /etc/confluent/docker/bash-config && \
+/etc/confluent/docker/configure && \
+/etc/confluent/docker/launch",
```

Comment:
This might be more readable if you use a raw-string literal because you can use actual line-breaks without escaping.

Reply:
Addressed in `81be19a`.

Reply:
Addressed in 81be19a.